instrumentation/spring/spring-jms-2.0/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,57 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+  id("org.unbroken-dome.test-sets")
+}
+
+muzzle {
+  pass {
+    group.set("org.springframework")
+    module.set("spring-jms")
+    versions.set("[2.0,)")
```

Comment:
2.0 seems to be the first version of spring-jms. Is the preferred way to still add the version number to module name even if the instrumentation is expected to cover the whole version range? I'm fine with either way, just asking.

Reply:
> Is the preferred way to still add the version number to module name even if the instrumentation is expected to cover the whole version range?

ya, I think so, in case we need to add `spring-jms-6.0` in the future, we don't need to rename the old one